cypress/e2e/tests/custom-metrics/analysis_metrics.test.ts

```diff
@@ -33,6 +28,11 @@ describe(["@tier2"], "Custom Metrics - Count the total number of initiated analy
 
         // Interceptors
         cy.intercept("GET", "/hub/application*").as("getApplication");
+
+        // Get the current counter value
+        metrics.getValue(metricName).then((counterValue) => {
+            counter = counterValue;
+        });
     });
 
     it("Perform analyses - Validate the tasks initiated counter increased", function () {
```

Comment:
Just a simple spelling mistake-`analysis `

Reply:
I think she used analyses for multiple analysis.

Reply:
@kpunwatk @sshveta correct, I meant to the plural verb